engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -318,7 +345,7 @@ public void preRenderSetupLightGeometry() {
         glDisable(GL_DEPTH_TEST);
 
         glEnable(GL_BLEND);
-        glBlendFunc(GL_ONE, GL_ONE);
+        glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR);
```

Comment:
I would suggest that thrown torches cannot be plugged in to the light propagation system as it would likely cause a CPU bottleneck.  Mainly because the chunk meshes would have to be constantly updated as the physics updated.

More accurate presentation of light photons would be desireable.  I perused a bunch of different ways global illumination could be more realistic,  but all were:
1. above my paygrade.
2. significantly would change shaderland.